boards/cc2538dk/Makefile.include

```diff
@@ -23,7 +23,7 @@ RESET ?= $(RIOTBOARD)/$(BOARD)/dist/reset.sh
 export PROGRAMMER ?= cc2538-bsl
 
 ifeq ($(PROGRAMMER),cc2538-bsl)
-  FLASHER = $(RIOTTOOLS)/cc2538-bsl/cc2538-bsl.py
+  FLASHER = $(RIOTTOOLS)/cc2538-bsl/cc2538-bsl.py -a $(IMAGE_OFFSET)+$(ROM_START_ADDR)
```

Comment:
This would need to go in `FFLAGS` instead of `FLASHER`.
Also, I would prefer with somehow only adding it only if `IMAGE_OFFSET` is set. Like in https://github.com/RIOT-OS/RIOT/blob/fc9577030e44b03aa279be21b4271f6b6d247c94/makefiles/tools/edbg.inc.mk#L14-L15
Even if `cc2538-bsl.py` handles if the `-a` is just +rom_start as it does eval.

Reply:
How does it behave if you remove the `-e` in `FFLAGS` ?

Reply:
I think it would be possible to assume the CCA is always there, and just work around that memory.

Reply:
I disabled the **-e** in **FFLAGS** so that i could flash the images to slots 0 and 1 individually. However it doesn't select the latest firmware. I posted my test results below. I'm not sure whether the issue is the linking of the bootloader, or the checksum